control/state.yaml

```diff
@@ -143,18 +150,6 @@ components:
           type: string
           enum: [withdraw, advertise]
 
-    Ping.Request:
```

Comment:
I'm curious as to why only 'Ping.Request' was in this file earlier?  The new location (control/ping.yaml) seems the right one to begin with.

Reply:
actually this file (control/state.yaml) was called `control/control.yaml` before, hence it was placed there to be consistent with other control API requests that were already there.